mantidimaging/core/io/test/filenames_test.py

```diff
@@ -184,6 +197,8 @@ def test_find_log_best(self):
         ("/a/Tomo/foo_Tomo_%06d.tif", "/a/Flat_Before/foo_Flat_Before_%06d.tif"),
```

Comment:
NITPICK: We should probably have a test following this structure to test invalid naming as well.

Reply:
Might need to wait until we have an example of something that we should not have found.